Cargo.toml

```diff
@@ -22,11 +22,10 @@ keywords = [ "simulation", "2d", "physics" ]
 myelin-geometry = "2.4"
 nalgebra = "0.18"
 ncollide2d = "0.19"
-serde = "1.0"
-serde_derive = "1.0"
+serde = { version = "1.0", features = ["derive"] }
 alga = "0.9"
 nameof = "1.0"
-mockiato = { version = "0.4", optional = true }
+mockiato = { git = "https://github.com/myelin-ai/mockiato", optional = true }
```

Comment:
Release mockiato and use the correct version

Reply:
Already in a todo